rero_ils/views.py

```diff
@@ -405,11 +405,26 @@ def nl2br(string):
     return string.replace("\n", "<br>")
 
 
+@blueprint.app_template_filter('urlActive')
+def url_active(string, target):
+    """Add link url on text if http detected."""
+    result = re.findall('(https?://[\\w|.|\\-|\\/]+)', string)
```

Comment:
probably you can use a regex to replace multiple values here.

Reply:
I don't understand. This code is not from me, and it is already a regex.